modules/cnf-working-with-fast-event-notifications-api.adoc

```diff
@@ -0,0 +1,283 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-working-with-fast-event-notifications-api_{context}"]
+= Working with the PTP fast event notifications REST API
+
+You can use the PTP fast event notifications REST API to subscribe an application to the PTP events that are generated for the parent node. PTP fast events notifications are available on each node where a PTP capable network interface is configured.
+
+You can subscribe DU applications to PTP event notifications using the resource address `/cluster/node/<node_name>/ptp`, where `<node_name>` is the cluster node running the DU application.
+
+Status request are sent by making a REST API `PUT` call to `/subscriptions/status/<subscription_id>`. The API call returns events through the AMQP event bus to the subscribed address.
+
+The PTP fast events notifications REST API is served at `http://localhost:8080/` by default. The following API endpoints are available:
+
+* `/api/cloudNotifications/v1/publishers`
+- `POST`: Creates a new publisher
+- `GET`: Retrieves a list of publishers
+* `/api/cloudNotifications/v1/publishers/<publisher_id>`
+- `GET`: Creates a new status ping request for the specified publisher id
+* `/api/cloudNotifications/v1/subscriptions`
+- `POST`: Creates a new subscription
+- `GET`: Retrieves a list of subscriptions
+* `/api/cloudNotifications/v1/subscriptions/<subscription_id>`
+- `GET`: Creates a new status ping request for the specified subscription id
+* `api/cloudNotifications/v1/subscriptions/status/<subscription_id>`
+- `PUT`: Creates a new status ping request for the specified subscription id
+* `/api/cloudNotifications/v1/health`
+- `GET`: Returns the health status of `cloudNotifications` API
+
+
+`cloudNotifications` API routes are described in more detail below.
```

Comment:
> ! below _adjective, adverb, preposition_
Do not use to refer to an earlier software version or fix level. Use "earlier".

Reply:
Removed the sentence. It's redundant.